src/renderer/components/+entity-settings/entity-settings.tsx

```diff
@@ -57,28 +58,40 @@ export class EntitySettings extends React.Component<Props> {
   async componentDidMount() {
     const { hash } = navigation.location;
 
-    this.ensureActiveTab();
+    if (hash) {
+      const item = this.menuItems.find((item) => item.title === hash.slice(1));
+
+      if (item) {
```

Comment:
Something that I have thought about from conversations with @leenamba. I think that the user should probably be warned (via a Notification or something) if an item is not found. 

That should help with spelling mistakes on `lens://` query parameters.